nano/node/lmdb/block_store.hpp

```diff
@@ -82,6 +82,12 @@ namespace lmdb
 		 */
 		MDB_dbi state_blocks_handle{ 0 };
 
+		/**
+		 * Unchecked bootstrap blocks info. (Removed)
+		 * nano::block_hash -> nano::unchecked_info
+		 */
+		MDB_dbi unchecked_handle{ 0 };
```

Comment:
Since this is remove we don't need it in the class definition anymore, it can be moved in to the upgrade path.